content/welcome.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Welcome
+---
+
+Welcome to the Meteor guide.
+
+This is a set of articles outlining opinions on best-practice application development in the [Meteor](https://meteor.com) framework. Our aim is that the set of articles detailed here covers patterns that are common to the development of all modern web and mobile applications. In this sense a lot of what's documented here is not necessarily Meteor specific and could be equally be applied to any application built with a focus on modern, reactive user interfaces.
+
+Nothing in the Meteor guide is *required* to build a Meteor application (i.e. you can certainly use the platform in ways that contradict the principles and patterns of the guide). However, the guide is an attempt to document what we consider are best practices for app development. The guide is also an effort to document community conventions, and so we expect that in the future the practices documented here will generally be followed by the majority of the Meteor community.
+
+To learn about "pure" Meteor, you can The APIs of the Meteor platform are available a the [docs site](https://docs.meteor.com), and you can browse community packages on [atmosphere](https://atmospherejs.com).
+
+The guide is intended as an intermediate document, aimed at those with some familiarity with the Meteor platform and web development in general. If you are just getting started with Meteor, we recommend beginning with the official [tutorial](https://www.meteor.com/tutorials/blaze/creating-an-app).
+
+<h2 id="example-app">Example App</h2>
+Most guides make reference to the Todos example application. You can create your own copy of the example application with
```

Comment:
Hmm, this won't be true at soft launch time; perhaps we should recommend that people clone it from GitHub? Also it kinda sucks that we can't update the example app frequently since it's baked into the Meteor tool...